package/Dockerfile

```diff
@@ -106,7 +106,7 @@ RUN curl -sLf ${!TINI_URL} > /usr/bin/tini && \
     mkdir -p /var/lib/rancher-data/driver-metadata
 
 ENV CATTLE_UI_VERSION 2.5.2-rc8.1
-ENV CATTLE_DASHBOARD_UI_VERSION v2.5.2-rc5
+ENV CATTLE_DASHBOARD_UI_VERSION release-2.6
```

Comment:
This will move to tags as soon as first tag is cut, and then continue to be tags (not revert to the release branch) until the release.